packages/web-components/@sfx/sayt/stories/index.ts

```diff
@@ -1,25 +1,48 @@
 import { storiesOf } from '@storybook/html';
 import { withKnobs, text, boolean } from '@storybook/addon-knobs';
+
 import '../src/index.ts';
+import { getDisplayCode } from '../../../../../.storybook/common';
```

Comment:
Do we not have a package-local util file that re-exports entities from `.storybook` so we don't have to go all the way up for them? If we can't have multiple story files, then ignore this suggestion.